### ISSUE TITLE
include text about N. vs. S. of Point Conception and why we didn't account for S. California spatial effects

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [iantaylor-NOAA/Lingcod_2021#46]
from John B.
```
South of Conception, growth as well as recruitment patterns may differ given the differences in ecological regimes in the
San Diegan vs Oregonian geographic provinces including water temperature, species compositions (competitors etc) and
primary productivity since there is not much upwelling down there.  I get that we don’t want to slice the pie too thin
or make extra work with excess areas, so if you are satisfied there is no major difference in depletion or dynamics
north and south of Conception, then ok, you just might want to cover that in your write up since they are different
biogeographic provinces within California. 
With work done by Harms, we now have much more data from Southern California to inform regional assessments,
so I thought it was worth broaching.  Consider it a box check consideration in the write up.
Not sure how further down the rabbit hole you have gone or want to go there.
If there is no good reason in what you can see, then so be it, but please just capture it since there may be questions.
```

**Comment by kellijohnson-NOAA** [comment]
@aaronmberger-nwfsc can you summarize the lack of spatially-explicit stock assessment models in a few sentences for me so we can rationalize that actually accounting for space in a production model right now is limited given our current workflow.

**Comment by aaronmberger-nwfsc** [comment]
Yes, sure thing.  I’m on the road currently, when do you need it by?

**Comment by kellijohnson-NOAA** [comment]
noon on 06-28-2021 because doc is due Monday the 28th

**Comment by aaronmberger-nwfsc** [comment]
Maybe some version of the text below could fit in there somewhere?  I also attached the word doc version of the text.  
[Spatial stock assessment Blurb.docx](https://github.com/iantaylor-NOAA/Lingcod_2021/files/6727482/Spatial.stock.assessment.Blurb.docx)
Let me know if you want something different.   I tried to be general enough but still applicable perhaps to decisions for Lingcod.

Spatial patterns in a stock’s population structure, and how those patterns are influenced by fishing and local environmental conditions, are important considerations for defining stock assessment and management unit boundaries (Cadrin et al. 2020; Berger et al. 2020).  Scientifically, there is growing appreciation and application of spatial stock assessment methods, primarily stemming from experimental settings (i.e., computer simulations). The results of this research underlines the importance of acknowledging spatial processes (e.g., connectivity dynamics between unique segments of a stock; biological characteristics that change across environmental gradients; and regulations that impose local changes in fishing patterns) across the management domain.  While the operationalization of management procedures that incorporate such complexities is now occurring for a few select (data rich) stocks globally, the widespread application of such approaches are not yet standard practice.  The main limitations arise from the added model complexity associated with increasing spatiotemporal dimensions, including underlying data limitations, confronting the expanding number of decisions or assumptions that need to be made, and the amount of analyst time required to develop, test, and vet spatial procedures given production stock assessment timelines (Berger et al. 2017; Punt 2019).


Berger, A.M., D.R. Goethel, P.D. Lynch, T. Quinn, S. Mormede, J. McKenzie, A. Dunn. Space oddity: the mission for spatial integration. Can. J. Fish. Aquat. Sci., 74 (2017), pp. 1698-1716.

Berger, A.M., J.J. Deroba, K.M. Bosley, D.R. Goethel, B.J. Langseth, A.M. Schueller, D.H. Hanselman. 2020. Incoherent dimensionality in fisheries management: consequences of misaligned stock assessment and population boundaries. ICES JMS 78, 155-171.

Cadrin, S.X, M.N. Maunder, A.E. Punt. 2020. Spatial structure: theory, estimation and application in stock assessment models. Fish Res 229. https://doi.org/10.1016/j.fishres.2020.105608

Punt, A. E. (2019b). Spatial stock assessment methods: A viewpoint on current issues and assumptions. Fisheries Research, 213, 132–143. https://doi.org/10.1016/j.fishr es.2019.01.014
